docs/faq/proof-of-stake/pos-general-faq.md

```diff
@@ -0,0 +1,123 @@
+# **<i class="fa fa-ticket"></i> PoS General FAQ**
+
+---
+
+#### **1. What is Decred's Proof-of-Stake system?**
+
+Besides Proof-of-Work (PoW) mining, Decred has a Proof-of-Stake (PoS) system. In this PoS system the participants buy tickets using their Decred coins (DCR). These tickets go into the PoS system ticket pool and can be called upon to vote on the previous block, like a lottery. There is a very small chance your ticket will not be called upon to vote.
+
+Every block 5 tickets are chosen from the PoS ticket pool. If at least 3 of those validate the previously mined block, the newly mined block is added to the chain, and both PoW miners and PoS stakers are paid. If the block is not validated by the PoS stakers the PoW miners do not get paid, but the PoS stakers do. This is to incentivize PoW miners to mine according to the wishes of the PoS stakers.
+
+This PoS system has several advantages that solve problems PoW-only systems might encounter. For example, because the PoS stakers have to validate the blocks the PoW miners mine, the PoW miners cannot decide on their own to change the rules of the network (a 51% attack). Or, when the Decred chain hardforks the old, smaller, chain will die off quickly due to not being validated by the PoS component of the chain.
+
+The PoS system rewards participants for their staking. Every Decred block has a PoS reward component that will be distributed to every ticket chosen to vote in that block. The system is designed in such a way that the average time for the ticket to be selected is 28 days. If a ticket hasn't been selected after 40960 blocks (about 4 months) it will be revoked by the system. The DCR used to purchase the ticket will be returned to the ticket owner.
+
+Another interesting feature this PoS system allows for is the ability to vote on an agenda. Because the block validation can be considered a vote, by adding extra votebit components to the ticket the system can track and count votes over a series of blocks. These votes can be about anything.
```

Comment:
i think this paragraph needs to be totally rewritten, it's just not clear enough as is.

For example:

> Besides voting on transactions in the previous block, the Decred PoS system can be used to vote on arbitrary decisions, including hard-forking consensus changes to the network.

This is a single sentence that covers essentially what this entire paragraph describes (without the technical details about bits that is not needed in this intro section) and also mentions how the system can be used for hardfork upgrades.